schemas/components/schemas/TamperingResult.yaml

```diff
@@ -5,7 +5,7 @@ properties:
     type: boolean
     description: >-
       Flag indicating whether browser tampering was detected according to our
-      internal thresholds.
+      internal thresholds or anti-detect browser was detected.
```

Comment:
I would suggest a clarified phrasing that includes some explanation of what anti-detect browsers are and an example. I just picked a top Google result, @xnerhu Can we in fact detect Octobrowser? 

> Flag indicating browser tampering was detected. This happens when: 
> * There are inconsistencies in the browser configuration that cross our internal tampering thresholds. 
> * The browser is one of the "anti-detect" browsers purposefully designed to evade identification and fingerprinting, for example, Octo Browser.